drivers/VDI.py

```diff
@@ -16,6 +16,7 @@
 # VDI: Base class for virtual disk instances
 #
 
+import cleanup
```

Comment:
Rather than importing cleanup, maybe at some point the constants you need from cleanup could be moved into their own file.

Reply:
Yes probably